File: example/src/main/java/com/mercadopago/android/px/addons/FakeLocaleBehaviourImpl.java
package com.mercadopago.android.px.addons;

import android.content.Context;
import android.support.annotation.NonNull;
import java.util.Locale;

public final class FakeLocaleBehaviourImpl implements LocaleBehaviour {

    @NonNull
    @Override
    public Context attachBaseContext(@NonNull final Context context) {
        final Locale locale = new Locale("en", "US");
        return LocaleContextWrapper.wrap(context, locale);
    }
}

File: example/src/main/java/com/mercadopago/android/px/addons/FakeLocaleBehaviourImpl.kt
package com.mercadopago.android.px.addons

import android.content.Context
import java.util.*

object FakeLocaleBehaviourImpl : LocaleBehaviour {
    var localeTag = "en-US"
    override fun attachBaseContext(context: Context): Context {
        val (language, country) = localeTag.split("-")
        return LocaleContextWrapper.wrap(context, Locale(language, country))
    }
}